### ISSUE TITLE
pcars.kick functionality?

### ISSUE THREAD

**Issue by TylerBre** [JcPires/Project-Cars-DS-LiveApp#1]
Noticed that you have a 'kick' method that makes a request to the pcars_ds http api. I can't find any documentation on this, is this even implemented?

https://github.com/JcPires/Project-Cars-DS-LiveApp/blob/fa1767ca6f0490a1250a44df7f8437069fd677ff/api/hooks/pcarsServer/index.js#L552

**Comment by JcPires** [comment]
I have to implement it. I'm waiting the next DS update to add all new features and this one.

**Comment by TylerBre** [comment]
can you programmatically kick a user via the http api though? that would be really convenient, rather than updating the blacklist and restarting the server

**Comment by JcPires** [comment]
Yes it could be possible with the SteamID and adding a " ban" field in the driver table. . At the connection , if the field is true , a kick event starts. It is possible!
